README.md

```diff
@@ -21,20 +19,121 @@
 * Can drag a View Controller onto the canvas.
 * Can locate and drag items from the Object Library.
 * Can locate the Attributes Inspector and change the Background color of Views
-* They know how to connect the ViewController.swift file and understand it's view life cycle.
+* They know how to connect the `ViewController.swift` file and understand its view life cycle.
 * Can hook up IBOutlets and IBActions.
 
-## Outline / Notes
+## UIKit
+
+iOS includes a framework called _UIKit_. UIKit provides you with a standard set of widgets, referred to as _elements_ or _controls_, that you can use in your app's views to provide a way for your users to interact with your app.
+
+## UI Controls
+
+So far, you have been introduced to a few user interface elements, including basic views, labels, and buttons. In this lesson, you'll get a quick overview of some of the other interesting user interface elements that you can use in your iOS apps.
+
+To begin this investigation, create a new iOS project in Xcode. You can use the single-view application template provided with Xcode. (There is also a sample application included in this repository. It's located at `UIElements/UIElements.xcodeproj`. But you should start your own project so you can get a feel for these elements yourself.)
+
+### Labels
+
+_Labels_, also known as `UILabel`s, are the most basic way to display strings of text in your app. You've already seen a few labels in your previous lessons, and should be pretty comfortable with them by now. Just for old time's sake, go ahead and drag a label onto your app's main view. You can find it in the object library in the bottom left corner of Xcode's main window. (You _do_ remember where that is and how to filter for labels, right?) Drop it in the top left corner of the application, so it looks like this:
+
```

Comment:
I think you mean "You can find it in the object library in the bottom right corner of Xcode's main window."